www/src/pages/index.js

```diff
@@ -134,7 +134,7 @@ class IndexRoute extends React.Component {
                   </Card>
 
                   <Diagram
-                    containerCSS={{
+                    containerCss={{
```

Comment:
Looking through the site it seems this components is only used here, so we could get rid of this prop and move these styles straight [into the component itself](https://github.com/gatsbyjs/gatsby/blob/034453645ceff61a1fd739816a0ef2cbf2f73f8c/www/src/components/diagram.js#L234).